be/tests/Feature/HTTP/ActivityLogsTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Tests\Feature\HTTP;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class ActivityLogsTest extends TestCase
+{
+    public function test_get_activity_logs_api()
+    {
+        $this->resposeChainBuilder('/api/activity_logs/');
+    }
+
+    public function test_get_activity_logs_for_specific_user()
+    {
+        $user = User::first();
+        $this->resposeChainBuilder('/api/activity_logs/' . $user->id);
+    }
+
+    private function resposeChainBuilder($uri, $method = null, $statusCode = null,)
+    {
+        $res = $this->withHeaders([
+            'Accept' => 'application/json',
+            'Content-type' => 'application/json',
+            'Authorization' => 'Bearer ' . env('API_TOKEN')
+        ])->json(
+            $method ?? "GET",
+            $uri
+        );
+        $res->assertStatus($statusCode ?? 200);
```

Comment:
- you should have separate test cases for both success and failing scenarios.
- follow proper format. its should have description.
- please read in this documentation how to properly write a test names (this is not laravel test). https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/
- assertion should not contain condition. since it is based in your expectation.
ex.
```
/**
 * @test
 * 
 * check if can successfully submit the answers to database
 */
public function it_should_submit_an_answer()
{
	$data = ...your logic here
	
	// you are expecting that there should be data inserted in the database
	$this->assertNotNull($data)
}


/**
 * @test
 * 
 * ... description here
 */
public function it_should_return_empty_if_data_is_invalid()
{
	// you know that id 99 not existed
	$data = ...

	// you are expecting an empty data
	$this->assertEmpty($data);
}
```